Light.json

```diff
@@ -4494,7 +4496,7 @@
         "type": "Character",
         "subType": "Droid",
         "uniqueness": "*",
-        "destiny": "Pi or 2Pi",
+        "destiny": "π or 2π",
```

Comment:
Please use the high number of `6.28318` as the destiny.
For the low number of `3.14159`, please place in a field of `uniqueDestiny: "3.14159"`